build.cmd

```diff
@@ -57,6 +57,7 @@ set __DoCrossgen=
 set __BuildSequential=
 set __msbuildCleanBuildArgs=
 set __msbuildExtraArgs=
+set __DisableOSSParam=
```

Comment:
I would rename this to something like "SignTypeReal" as our intent is to enable real-signing in the build.